### DISCUSSION TITLE
Users in... Contains or Has $CURRENT_USER?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15858] [Feature Requests]
As `$CURRENT_USER` in filters does not support properties, it would be nice to have the reverse of `one of` for, let say, `Users in role`
```
{
	"_and": [
		{
			"users": {
				"_contains": [
					"$CURRENT_USER"
				]
			}
		}
	]
}
```
`Contains` or `Has` or ...

**Comment by Pachat** [comment]
I found a solution with a condition on the junction table (here `junction_directus_users_directus_roles`) instead.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
